hack/jenkins/e2e.sh

```diff
@@ -443,7 +443,7 @@ case ${JOB_NAME} in
     : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
     : ${E2E_CLUSTER_NAME:="jkns-gke-e2e-test"}
     : ${E2E_NETWORK:="e2e-gke-test"}
-    : ${JENKINS_USE_RELEASE_TARS:=y}
+    : ${JENKINS_PUBLISHED_VERSION:='release/latest'}
```

Comment:
minor nit, but every other string here uses " instead of ' - is there a particular reason to be different?